cmd/mindthegap/flags/validate.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2023 D2iQ, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package flags
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+func ValidateRequiredFlagValues(cmd *cobra.Command, requiredFlagsWithValues ...string) error {
+	fs := cmd.Flags()
+
+	missingFlagValues := []string{}
+	for _, flagName := range requiredFlagsWithValues {
+		foundFlag := fs.Lookup(flagName)
+		if foundFlag == nil {
```

Comment:
nit: if the flag is provided or not is already validated using `ValidateRequiredFlags` . since we ignore the flag that is not found here, should we rename method to `ValidateFlagValues` ?  wdyt?

Reply:
`fs.Lookup(flagName)` looks up the flag in the flagset. It is not about whether it is provided or not, just whether it even exists as declared as part of the command definition. Skipping here will only happen if the flag doesn't exist at all, and can only happen as part of a misconfiguration in the command definition. I think it's ok to skip here.

I'll rename the function to `ValidateFlagsThatRequireValues`.